shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -2034,6 +2035,41 @@ export default {
     handlePsaDefaultChanged() {
       this.togglePsaDefault();
     },
+    handleMachinePoolError(error) {
```

Comment:
I'm thinking that I might move this logic to `shell/machine-config/vmwarevsphere.vue` so that all of the vsphere error logic is handled in a single place.

Reply:
I'm opting to keep this as is for now. We can follow up at a later date if we want this to change.